storybook/storybook-react/src/components/Heading/Heading.stories.tsx

```diff
@@ -4,55 +4,53 @@
  */
 
 import { Heading } from '@amsterdam/design-system-react/src'
-import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { Meta } from '@storybook/react'
 
 import '@amsterdam/design-system-css/src/heading/heading.scss'
 
 export default {
   title: 'Heading',
   id: 'heading',
   component: Heading,
-} as ComponentMeta<typeof Heading>
+} satisfies Meta<typeof Heading>
 
-const Template: ComponentStory<typeof Heading> = (args) => <Heading {...args} />
-
-export const Heading1 = Template.bind({})
-
-Heading1.args = {
-  children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+export const Heading1 = {
+  args: {
+    children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+  },
 }
 
-export const Heading2 = Template.bind({})
-
-Heading2.args = {
-  children: 'Jouw typograaf biedt mij zulke exquise schreven!',
-  level: 2,
+export const Heading2 = {
+  args: {
+    children: 'Jouw typograaf biedt mij zulke exquise schreven!',
```

Comment:
This can be a bit more DRY: 

```suggestion
    ...Heading1.args,
```

Same for the other heading levels